app/components/ViewContainer.js

```diff
@@ -1,19 +1,50 @@
 import React, { Component, PropTypes, View } from 'react-native';
 import AR from './AR.js';
 import Map from './Map.js';
+import DropNewPinButton from '../containers/container_dropNewPin'
 
 export default class ViewContainer extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      view: 'map'
+      view: 'map',
+      currLoc: null,
     };
+    this.fireRef = new Firebase("https://interruptedlobster.firebaseio.com/");
   }
+
+  componentDidMount() {
```

Comment:
Will this update the data in realtime as we walk around, or will it only fire once when the component is mounted to the DOM?